packages/remix-oxygen/src/server.ts

```diff
@@ -7,19 +7,27 @@ import {
 export function createRequestHandler<Context = unknown>({
   build,
   mode,
+  poweredByHeader = true,
   getLoadContext,
 }: {
   build: ServerBuild;
   mode?: string;
+  poweredByHeader?: boolean;
   getLoadContext?: (request: Request) => Promise<Context> | Context;
 }) {
   const handleRequest = createRemixRequestHandler(build, mode);
 
   return async (request: Request) => {
-    return handleRequest(
+    const response = await handleRequest(
       request,
       (await getLoadContext?.(request)) as AppLoadContext,
     );
+
+    if (poweredByHeader) {
+      response.headers.append('Powered-By', 'Shopify-Hydrogen');
```

Comment:
I'm curious how you came to this specific header? When I look at the linked issue it has it as a different format, e.g. `Shopify, Oxygen, Hydrogen`

I'm also under the impression that Oxygen will add `Shopify, Oxygen`?

Reply:
Yeah +1, according to this ticket where Tobi made the request: https://github.com/Shopify/oxygen-platform/issues/726

The request was `powered-by: Shopify, Hydrogen` — I don't know if Oxygen was actually requested anywhere, but would be good to follow the requested semantics if there's no big issue with it. `Shopify-Hydrogen` was already in place at that time, and this was specifically asked for.

Reply:
The Oxygen folks were talking about adding another "Oxygen" one on their end.